### PR TITLE
Add getFile method on message attachment resource

### PR DESCRIPTION
**Pull request by cdupoiron**
Add getFile method on message attachment resource in order to download attachment with BODY+HEADER response, example:

ctxioClient.users(config.contextio.userId).email_accounts(config.contextio.label).folders(folder).messages(messageId).attachments(attachmentId).getFile().then(function (ctxioClientRes) {
    res.set(ctxioClientRes.headers);
    res.send(ctxioClientRes.body);
  });

### CODE REVIEW

File: lib/lite/index.js [hunk 1]
```diff
@@ -88,7 +88,10 @@ module.exports = function (settings) {
                       resource += attachment_id ? 'attachments/' + encodeURIComponent(attachment_id) + '/' : 'attachments/'
 
                       return {
-                        get: _callService.get.bind(_callService, resource)
+                        get: _callService.get.bind(_callService, resource),
+                        getFile: function() {
```
Comment (AaronLaBrie) [inline lib/lite/index.js]: For consistency's sake, I'd like this to use the `bind` syntax here.

I think `getFile: _callService.getFile.bind(_callService, resource)` should do the trick

**Comment by cdupoiron** [comment]
Done :)

**Comment by AaronLaBrie** [comment]
Thanks for contributing! I'll have this out shortly.

**Comment by AaronLaBrie** [comment]
Released + Published.

**Pull request merged**